modules/dxp/apps/digital-sales-room/digital-sales-room-web/test/js/utils/Utils.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import '@testing-library/jest-dom';
+import {act, fireEvent} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+export async function setFieldValue(
```

Comment:
[NB] Evaluate renaming the file as `index.ts` as it's the only one present in your test util directory.